src/axom/quest/examples/containment_driver.cpp

```diff
@@ -415,10 +415,10 @@ class ContainmentDriver
     if(dimension() == 3)
     {
       fmt::memory_buffer edgeHistStr;
-      fmt::format_to(edgeHistStr, "Edge length histogram (lg-arithmic): ");
+      fmt::format_to(std::back_inserter(edgeHistStr), "Edge length histogram (lg-arithmic): ");
```

Comment:
Thanks for the fix. 

The new way seems awkward though... 
Perhaps we should move away from  `format_to` and `memory_buffer`? (in a follow-up PR).

I'm not sure if this is best practice, but something like the following should work:
```cpp
int a = ...;
double b = ...;
std::stringstream sstr;
fmt::print(sstr, "format string {} {}", a, b);
```
And is a lot cleaner (not sure about the efficiency)

Reply:
Yea i am fine with whatever this was just what was the first fmt example i found showed